data/create_users

```diff
@@ -5,5 +5,5 @@ encrypted_password="$2"
 
 for i in `seq 1 $n_users` ; do
 	num=`printf "%02d" $i`
```

Comment:
now it looks like there is inconsistent style in this file. This line '\t', the one you changed with four spaces. Please add another commit which uses 4 spaces consistently